Lib/base64.py

```diff
@@ -299,29 +299,61 @@ def b16decode(s, casefold=False):
 _A85START = b"<~"
 _A85END = b"~>"
 
+def _85buffer_iter_words(b):
+    # Utility method for _85encode
+    # yield unpacked int32 words from buffer, hopefully in an efficient manner,
+    # padding the last part with NULL bytes if necessary
+    n1 = len(b) // 512  # number of 512 bytes unpack
+    n2 = (len(b) - n1 * 512) // 4  # number of 4 bytes unpack
+    padding = (-len(b)) % 4
+
+    unpack512 = struct.Struct("!128I").unpack
+    unpack4 = struct.Struct("!I").unpack
+
+    offset = 0
+    for _ in range(n1):
+        for c in unpack512(b[offset:offset+512]):
+            yield c
```

Comment:
This can be simplified to:
```suggestion
        yield from unpack512(b[offset:offset+512])
```

Reply:
Changed (I'm not yet used to yield from ^^)